### PR TITLE
Add `isNode` to react-is

### PR DESCRIPTION
**Pull request by jquense**
Wanted this today and noticed it didn’t exist, What do ya’ll think? It adds some parity with prop-types although i did remove the iterator support to keep is small and i think the support there in react is still bit up in the air?

### CODE REVIEW

**Comment by react-linter** [comment]
<details>
  <summary>Details of bundled changes.</summary>

  <p>Comparing: 3297102de6692a675aad239061eed7aa822115fe...efe8e670af1ee930447d169795cabbcbeabb8b1a</p>


  
## react-is
File | Filesize Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip | ENV
 ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  --- 
react-is.development.js | +8.1% | +8.3% | 4.67 KB | 5.05 KB | 1.29 KB | 1.4 KB | UMD_DEV
**react-is.production.min.js** | **:small_red_triangle:+10.6%** | **:small_red_triangle:+10.0%** | **1.89 KB** | **2.09 KB** | **787 B** | **866 B** | **UMD_PROD**
react-is.development.js | +8.4% | +8.9% | 4.49 KB | 4.87 KB | 1.23 KB | 1.34 KB | NODE_DEV
**react-is.production.min.js** | **:small_red_triangle:+11.2%** | **:small_red_triangle:+12.1%** | **1.84 KB** | **2.04 KB** | **719 B** | **806 B** | **NODE_PROD**
ReactIs-dev.js | +8.3% | +8.8% | 4.56 KB | 4.94 KB | 1.26 KB | 1.37 KB | FB_WWW_DEV
**ReactIs-prod.js** | **:small_red_triangle:+10.3%** | **:small_red_triangle:+11.0%** | **3.67 KB** | **4.04 KB** | **999 B** | **1.08 KB** | **FB_WWW_PROD**

  </details>
  
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: packages/react-is/src/ReactIs.js [hunk 1]
```diff
@@ -90,6 +90,23 @@ export function isForwardRef(object: any) {
...
+    case 'boolean':
+      return !object;
+    case 'object':
+      if (Array.isArray(object)) {
+        return object.every(isNode);
+      }
+      return object === null || isElement(object);
+    default:
+      return false;
+  }
```
Comment (nhunzaker) [inline packages/react-is/src/ReactIs.js]: Is it worth calling out Symbol in some way?

Comment (jquense) [reply packages/react-is/src/ReactIs.js]: seems like no? you really can't render them right?

Comment (nhunzaker) [reply packages/react-is/src/ReactIs.js]: Ah... right! Returning false as a default is probably sufficient.

**Pull request closed without merging**